app/web_framework/cluster_master_application.py

```diff
@@ -60,11 +60,27 @@ def initialize(self, route_node=None, cluster_master=None):
         super().initialize(route_node)
 
 
-class _RootHandler(_ClusterMasterBaseHandler):
+class _ClusterMasterBaseStaticFileHandler(ClusterBaseHandler, tornado.web.StaticFileHandler):
```

Comment:
Your original comment was very clear. I have no idea why I missed it. :disappointed: